cucumber/authenticators_gcp/features/authn_gcp.feature

```diff
@@ -0,0 +1,41 @@
+Feature: GCP Authenticator - Hosts can authenticate with GCP authenticator
+
+  In this feature we define a Google Cloud Platform authenticator in policy and perform authentication
+  with Conjur, using a host with account-service-id.
+  In successful scenarios we will also define a variable and permit the host to
+  execute it, to verify not only that the host can authenticate with the GCP
+  Authenticator, but that it can retrieve a secret using the Conjur access token.
+
+  Background:
+    Given a policy:
+    """
+    - !policy
+      id: conjur/authn-gcp
+      body:
+      - !webservice
+
+      - !group apps
+
+      - !permit
+        role: !group apps
+        privilege: [ read, authenticate ]
+        resource: !webservice
+    """
+    And I am the super-user
+    And I have host "test-app"
+    And I grant group "conjur/authn-gcp/apps" to host "test-app"
+
+  Scenario: Hosts can authenticate with GCP authenticator and fetch secret
+    Given I have a "variable" resource called "test-variable"
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
+    And I set authn-gcp/service-account-id annotation to host "test-app"
```

Comment:
do you want to add all 4 annotations? i guess we will later have a test for each but as this is the general vanilla flow i think we should have all 4.